### DISCUSSION TITLE
CORS- and sha-issues on cloudflare pages with auto minify turned on

### DISCUSSION THREAD

**Discussion by metasikander** [adityatelange/hugo-PaperMod#1041] [Q&A] [answered]
I just set up my site on cloudflare pages, and started to have some issues with the hash for css-files being wrong on my main site (worked fine on dev-address).
I got it resolved through the cloudflare community (https://community.cloudflare.com/t/cors-blocked-on-custom-domain-pages/417441), by disabling auto minify on my site (auto minify compresses js, css and/or html-files).
In the thread on cloudflare community i got a recomendation to minify my resources with this code
```html
{{ with resources.Get "assets/css/stylesheet.css" }}
	{{ $CSS := resources.Get "assets/css/stylesheet.css" }}
	{{ $CSS := $CSS | resources.Minify }}
	{{ $hash_CSS := $CSS | resources.Fingerprint "sha256" }}
	<link rel="preload stylesheet" href="{{ $hash_CSS.RelPermalink }}" integrity="{{ $hash_CSS.Data.Integrity }}" as="style" type="text/css" crossorigin="anonymous">
{{ end }}
```
I'm not a dev, so not sure how to do this, and i wonder if it's something that could be implemented in the code upstream.  
Any recomendations?

**Comment by adityatelange** [comment, marked as answer]
All assets css and js are already minified that is compressed. It's okay to disable asset minification on cloudflare. Won't impact anything.

**Comment by metasikander** [comment]
okay. great!
